docs/build/windows.md

```diff
@@ -314,75 +318,63 @@ choco install wixtoolset --version=3.11.1
 git clone https://github.com/keymanapp/CEF4Delphi_Binary C:\Projects\keyman\CEF4Delphi_Binary
 ```
 
-## Certificates
-
-In order to make a release build, you need to sign all the executables. See
-[windows/src/README.md#Certificates](../../windows/src/README.md#Certificates)
-for details on how to create test code signing certificates or specify your own
-certificates for the build.
-
-## Notes on Environment Variables
-
-### KEYMAN_ROOT
+### Android dependencies
 
-If you pull the entire `keyman.git` repo to `c:\keyman`, then the paths by
-default will work without changes. Otherwise, you will need to set an
-environment variable `KEYMAN_ROOT` to the root path of the Keyman repo. For
-example:
-
-```bat
-SET KEYMAN_ROOT=c:\projects\keyman\keyman
-```
-
-### KEYMAN_CEF4DELPHI_ROOT
+**Projects**:
+* Keyman for Android
 
-Keyman and Keyman Developer use Chromium Embedded Framework. The source repo is
-at https://github.com/keymanapp/CEF4Delphi. In order to build the installers, we
-need to source the binary files from the
-https://github.com/keymanapp/CEF4Delphi_binary repo. The
-`KEYMAN_CEF4DELPHI_ROOT` environment variable should be set to the root of this
-repo on your local machine.
+**Requirements**:
+* Android SDK
+* Android Studio
+* Ant
+* Gradle
+* Maven
+* JDK 11 (Temurin11)
 
-The version of CEF in use is determined by CEF_VERSION.md. This maps to a branch
-prefixed with `v` e.g. `v89.0.18` in the CEF4Delphi_binary repository. During a
-release build, the common/windows/cef-checkout.sh script will checkout the correct
-branch of the repository automatically and extract any compressed files found in
-it.
+#### JDK 11
 
-### GIT_BASH_FOR_KEYMAN
```

Comment:
We're dropping this section?  Do we not need the separate callout from batch builds anymore?

Reply:
The need has gone away with the move from Makefile to build.sh. There are a couple of lingering references in `gosh` but generally `gosh` figures it out without need for the variable, and `gosh` itself is basically only used in npm scripts, which we try not to use directly (because they are so slow to instantiate).

Reply:
I suppose there _is_ that, but there is the matter of the primary scripts themselves being shell scripts now.  Granted, I guess that'd mean directly using the shell, rather than having a different script establish it later.

... maybe I missed it, but do we explicity mention a required version of `bash` / `sh` / other shell to use for Windows shell-scripts?

Reply:
Yeah good point, the building docs for Windows are still out of date and reference nmake. Another chore, perhaps an ✈️ task?

Reply:
Follow-up in #12270